src/components/container/container/container.scss

```diff
@@ -1,5 +1,16 @@
 @use '../../core/styles' as sbb;
 
+@mixin container-background-color {
```

Comment:
I'm not convinced a mixin/repetition is the best solution here.
Is it not possible to create a rule that covers these two cases?